test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -248,12 +253,18 @@ func (sp *ScoreWithNormalizePlugin) Name() string {
 
 // reset returns name of the score plugin.
 func (sp *ScoreWithNormalizePlugin) reset() {
+	sp.mutex.Lock()
```

Comment:
The method will be removed in https://github.com/kubernetes/kubernetes/pull/116634, but I think the conflict is easy to solve.